src/util/ReactUtils.ts

```diff
@@ -282,18 +207,22 @@ export const isValidSpreadableProp = (
   property: unknown,
   key: string,
   includeEvents?: boolean,
-  svgElementType?: FilteredSvgElementType,
+  svgElementType?: SVGElementType,
 ) => {
   /**
    * If the svg element type is explicitly included, check against the filtered element key map
    * to determine if there are attributes that should only exist on that element type.
    * @todo Add an internal cjs version of https://github.com/wooorm/svg-element-attributes for full coverage.
    */
-  const matchingElementTypeKeys = FilteredElementKeyMap?.[svgElementType] ?? [];
+  const svgElementAttributesInCamelCase = getSVGElementAttributesInCamelCase(svgElementAttributes);
```

Comment:
the props we accept are also in camel case for the same reason (react), why do we need to do any transformation here?

Reply:
I created the `getSVGElementAttributesInCamelCase` cause the `svgElementAttributes` object structured as follows. I should have simply create the util to return the converted object without having to do the conversion multiple times. 
```js
circle: [
    'alignment-baseline',
    'baseline-shift',
    'clip',
    'clip-path',
    'clip-rule',
    'color',
    ...
    ]
    ```

Reply:
You can probably do the conversion and once and keep it stored as camel case, we don't need to do it at runtime